platform/darwin/src/MGLSymbolStyleLayer.h

```diff
@@ -1549,6 +1575,32 @@ MGL_EXPORT
  */
 @property (nonatomic, null_resettable) NSExpression *textTransform;
 
+/**
+ To increase the chance of placing high-priority labels on the map, you can
+ provide an array of `textAnchor` locations: the render will attempt to place
+ the label at each location, in order, before moving onto the next label. Use
+ `textJustify: auto` to choose justification based on anchor position. To apply
+ an offset, use the `textRadialOffset` instead of the two-dimensional
+ `textOffset`.
+ 
+ This property is only applied to the style if `textAnchor` is set to `nil`, and
+ `textOffset` is set to `nil`, and `symbolPlacement` is set to an expression
+ that evaluates to or `MGLSymbolPlacementPoint`. Otherwise, it is ignored.
+ 
+ You can set this property to an expression containing any of the following:
+ 
+ * Constant array values
```

Comment:
If this is accepts `MGLTextAnchor` values, the documentation should be similar to:
https://github.com/mapbox/mapbox-gl-native/blob/0ec89454503befbbfe628c1677955e23911e75c7/platform/darwin/src/MGLSymbolStyleLayer.h#L1145-L1159